helpers/github.ts

```diff
@@ -239,16 +235,32 @@ export function getSocialMediaText(issue: GitHubIssue): string {
   return `${priceLabel} for ${timeLabel}\n\n${issue.body}`;
 }
 
-export async function getProjectUrls() {
+export async function getProjectUrls(opts: typeof opt = opt) {
```

Comment:
"opts" generally implies options like parameters for a function which can be misleading. I would stick with a derivative of "opt"